CMake/RealmCore.cmake

```diff
@@ -342,10 +378,15 @@ endfunction()
 
 function(clone_and_build_realm_sync branch)
     set(cmake_files ${CMAKE_CURRENT_BINARY_DIR}${CMAKE_FILES_DIRECTORY})
+    if(REALM_PLATFORM STREQUAL "Android")
+        set(config_cmd test -f src/config.mk || REALM_CORE_PREFIX=${cmake_files}/realm-core/src/realm-core REALM_FORCE_OPENSSL=YES REALM_ENABLE_ENCRYPTION=YES REALM_ENABLE_ASSERTIONS= sh build.sh config && echo "ENABLE_ENCRYPTION    = yes" >> src/config.mk)
```

Comment:
Can you clarify what uses `ENABLE_ENCRYPTION`? I see nothing in realm-sync that does. AFAICT `build.sh build-android` unconditionally [downloads](https://github.com/realm/realm-sync/blob/7b311c1bc19388608e52e984f640ed7f5c98b114/build.sh#L808) and [builds](https://github.com/realm/realm-sync/blob/7b311c1bc19388608e52e984f640ed7f5c98b114/build.sh#L872-L899) OpenSSL. Are you perhaps thinking of realm-core, which does use `ENABLE_ENCRYPTION`?